python/sglang/test/test_utils.py

```diff
@@ -1837,3 +1840,34 @@ def wrapper(self):
         return wrapper
 
     return decorator
+
+
+def get_gpu_rank():
```

Comment:
The function is getting device count, not the current rank or device index. I feel like it's better to name the function to something like `get_gpu_count` and rename the variable from `gpu_rank` to `gpu_count`.